src/watchers/run.py

```
@@ -1,96 +1,157 @@
 
-    producer = Producer({"bootstrap.servers": kafka_bootstrap})
+    logger.info(
+        "watcher started — %d instance(s): %s",
+        len(instances),
+        ", ".join(i.id for i in instances),
+    )
 
-    running = True
+    stop_event = asyncio.Event()
 
     def handle_signal(signum, frame):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `handle_signal`

Add return type annotation: `None`

(ANN202)